cmd/git/main.go

```diff
@@ -401,6 +401,11 @@ func clone(ctx context.Context) error {
 		return err
 	}
 
+	safeDirectoryArgs := []string{"config", "--global", "--add", "safe.directory", flagValues.target}
+	if _, err := git(ctx, safeDirectoryArgs...); err != nil {
+		return err
+	}
```

Comment:
In unit tests, the Git command runs on your local machine. We therefore want to prevent the use of `git config --global`. The `safe.directory` configuration is therefore set in the `git` function itself.

Is the CLI really running on the latest Build code and somehow causes a Git operation to fail with this? If so, then we must look at this specific command. Though, I somehow remember that the CLI runs the latest release in e2e - which would be v0.11? We can have a chat, I wonder if using the latest nightly somehow in the CLI would fix things.